src/context.ts

```diff
@@ -544,7 +545,7 @@ export class Context implements RenamedUpdate {
     }
 
     /**
-     * Context-aware alias for `api.copyMessage`. Use this method to copy messages of any kind. Service messages and invoice messages can't be copied. The method is analogous to the method forwardMessage, but the copied message doesn't have a link to the original message. Returns the MessageId of the sent message on success.
+     * Context-aware alias for `api.copyMessage`. Use this method to copy messages of any kind. Service messages and invoice messages can't be copied. A quiz poll can be copied only if the value of the field correct_option_id is known to the bot. The method is analogous to the method forwardMessage, but the copied message doesn't have a link to the original message. Returns the MessageId of the sent message on success.
```

Comment:
Do we need back tick for `correct_option_id`?

Reply:
No. As stated in point one [here](https://github.com/grammyjs/types#differences-to-the-bot-api), we don't manually add formatting to the parts of the docs that are copied from the website. (At least not yet.)

This is only done for manually written docs, such as the prefixing sentence at the beginning of this line.